File: problemas/1000/1008.cs
using System;

namespace uri1008
{
    internal static class Program
    {
        private static void Main()
        {
            int numero = int.Parse(Console.ReadLine());
            int horas = int.Parse(Console.ReadLine());
            double valor = double.Parse(Console.ReadLine());
            Console.WriteLine($"NUMBER = {numero}");
            Console.WriteLine($"SALARY = U$ {(horas * valor):F2}");
        }
    }
}

File: problemas/1000/1008.dart
import 'dart:io';

void main() {
  final numero = int.parse(stdin.readLineSync());
  final horas = int.parse(stdin.readLineSync());
  final valor = double.parse(stdin.readLineSync());
  stdout.writeln('NUMBER = $numero');
  stdout.writeln('SALARY = U\$ ${(horas * valor).toStringAsFixed(2)}');
}